kumascript/macros/CSSRef.ejs

```diff
@@ -696,32 +765,35 @@ const text = mdn.localStringMap({
         'Backgrounds_and_borders': 'Backgrounds and borders',
         'Handling_different_text_directions': 'Handling different text directions',
         'Overflowing_content': 'Overflowing content',
-        'Values_and_units': 'Values and units',
+        'CSS_values_and_units': 'CSS values and units',
         'Sizing_items_in_CSS': 'Sizing items in CSS',
         'Images_media_and_form_elements': 'Images, media, and form elements',
         'Styling_tables': 'Styling tables',
         'Debugging_CSS': 'Debugging CSS',
         'Organizing_your_CSS': 'Organizing your CSS',
+        'Assessment_Fundamental_CSS_comprehension': 'Assessment: Fundamental CSS comprehension',
+        'Assessment_Creating_fancy_letterheaded_paper': 'Assessment: Creating fancy letterheaded paper',
+        'Assessment_A_cool_looking_box': 'Assessment: A cool-looking box',
       'Styling_text': '樣式化文字',
         'Styling_text_overview': '樣式化文字概述',
         'Fundamental_text_and_font_styling': '基礎的文字與字型樣式化',
         'Styling_lists': '樣式化列表',
         'Styling_links': '樣式化連結',
         'Web_fonts': 'Web 字型',
-        'Assessment_typesetting_a_community_school_homepage': '親合度：設定社區大學首頁的版面',
+        'Assessment_Typesetting_a_community_school_homepage': '親合度：設定社區大學首頁的版面',
       'CSS_layout': 'CSS 版面配置',
         'CSS_layout_overview': 'CSS 版面配置概述',
         'Normal_Flow': 'Normal Flow',
         'Flexbox': '彈性區塊',
         'Grids' : '格線',
         'Floats': '浮動',
         'Positioning': '定位',
-        'Multiple-column_Layout': 'Multiple-column Layout',
+        'Multiple-column_layout': 'Multiple-column layout',
         'Responsive_design': 'Responsive design',
-        'Media_queries': 'Beginner\'s guide to media queries',
-        'Legacy_Layout_Methods': 'Legacy Layout Methods',
-        'Supporting_Older_Browsers': 'Supporting Older Browsers',
-        'Fundamental_Layout_Comprehension': 'Fundamental Layout Comprehension',
+        'Beginners_guide_to_media_queries': 'Beginner\'s guide to media queries',
+        'Legacy_layout_methods': 'Legacy layout methods',
+        'Supporting_older_browsers': 'Supporting older browsers',
+        'Assessment_Fundamental_Layout_Comprehension': 'Assessment: Fundamental Layout Comprehension',
```

Comment:
@mdn/yari-content-zh Could you please review these `zh-TW` changes?

Reply:
/cc @irvin